src/app/containers/ConsentBanner/index.jsx

```diff
@@ -1,10 +1,36 @@
-import React, { useContext } from 'react';
+import React, { useContext, useCallback } from 'react';
 import { RequestContext } from '../../contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
+import { EventContext } from '../../contexts/EventContext';
+import { buildATIParams } from '../ATIAnalytics/params';
+import sendBeacon from '../../lib/analyticsUtils/sendBeacon';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+export const ConsentBanner = props => {
+  const requestContext = useContext(RequestContext);
+  const serviceContext = useContext(ServiceContext);
+  const eventContext = useContext(EventContext);
+
+  const query = buildATIParams(props, requestContext, serviceContext);
+  const atiUrl = process.env.SIMORGH_ATI_BASE_URL + query;
+
+  const { useClickTracker } = eventContext;
+
+  useClickTracker(
```

Comment:
Here's where the consent banner captures clicks on itself, and logs impressions on ATI.